docs/SingleNodeOpenShift.md

```diff
@@ -0,0 +1,29 @@
+# Single Node OpenShift
+
+Single Node OpenShift (SNO) clusters have a single control-plane node and are not considered HA, whereas OpenShift clusters with multiple control-plane nodes are considered HA as control-plane functions can be rescheduled on another node.
+
+## Differences
+
+There are several key differences in how the Machine Config Daemon handles non-HA/single node clusters:
+
+### Topology
+
+An annotation (`machineconfiguration.openshift.io/controlPlaneTopology`) is applied to all Node objects to describe the topology of the cluster. A non-HA/single node cluster is annotated with `SingleReplica`. In the absence of this annotation, the Machine Config Daemon defaults to `HighlyAvailable`. This primarily affects the node drain behavior as discussed below.
+
+### Node Drain Behavior
+
+In a non-HA/single node cluster, a node drain will **not** occur. This differs from HA clusters, where a [node drain](./MachineConfigDaemon.md#node-drain) will occur whenever required. 
+
+### Authentication and API Availability
+
+In a non-HA cluster, all authentication and API services will be unavailable until the node finishes rebooting. If those services do not come back up due to a failed configuration change or update, debugging may require SSH or console access to the node. This is because the OpenShift web UI or CLI (`oc`) may be unavailable. 
```

Comment:
Let's remove line `This is because the OpenShift web UI or CLI (oc) may be unavailable.` as reason of unreachable API can be anything that may not be MCO specific.